<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FivetigerClass</class>
 <widget class="QMainWindow" name="FivetigerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fivetiger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>130</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>先手方得分：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFGoal">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>后手方得分：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDGoal">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>当前走棋方：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblcurrentFlag">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>走棋阶段：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPhrase">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>对战电脑</string>
     </property>
     <addaction name="acEasy"/>
     <addaction name="acMid"/>
    </widget>
    <addaction name="acToSelfPlay"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="acToSelfPlay">
   <property name="text">
    <string>单人研究</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>五虎棋说明</string>
   </property>
  </action>
  <action name="acEasy">
   <property name="text">
    <string>简单难度</string>
   </property>
  </action>
  <action name="acMid">
   <property name="text">
    <string>中等难度</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Fivetiger.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acToSelfPlay</sender>
   <signal>triggered()</signal>
   <receiver>FivetigerClass</receiver>
   <slot>startToSelfClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acEasy</sender>
   <signal>triggered()</signal>
   <receiver>FivetigerClass</receiver>
   <slot>startToEasyClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action</sender>
   <signal>triggered()</signal>
   <receiver>FivetigerClass</receiver>
   <slot>helpClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acMid</sender>
   <signal>triggered()</signal>
   <receiver>FivetigerClass</receiver>
   <slot>startToMidClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startToEasyClick()</slot>
  <slot>startToSelfClick()</slot>
  <slot>helpClick()</slot>
  <slot>startToMidClick()</slot>
 </slots>
</ui>
